circle/main.go

```diff
@@ -112,6 +112,15 @@ func doDownload(flags *flag.FlagSet) error {
 
 func main() {
 	waitflags := flag.NewFlagSet("wait", flag.ExitOnError)
+	waitflags.Usage = func() {
+		fmt.Fprintf(os.Stderr, `usage: wait
+
+Wait for builds to complete, then print a descriptive output on 
+success or failure.
+`)
+		waitflags.PrintDefaults()
```

Comment:
No but if you add flags to the command, this will cause them to show up automatically